app/mailers/user_mailer.rb

```diff
@@ -37,7 +37,7 @@ def first_upload_reminder(submission:, user:)
             unique_args: {
               submission_id: submission.id
             }
-    mail to: user.email, subject: "You're Almost Done"
+    mail to: submission.email, subject: "You're Almost Done"
   end
 
   def second_upload_reminder(submission:, user:)
```

Comment:
since none of the functions use `user.email `anymore in this file, do you think it'd make sense to remove `user:` param from those functions?